SSZipArchive/Common.h

```diff
@@ -0,0 +1,81 @@
+#ifndef SSZipCommon
```

Comment:
This file looks like a modification of https://github.com/samsoffes/ssziparchive/blob/master/minizip/unzip.h.

Please follow OSS license requirements.